app/models/duplicate_checker_job_run.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class DuplicateCheckerJobRun < ApplicationRecord
+  has_many :potential_duplicate_people, dependent: :destroy
+  belongs_to :competition
+
+  default_scope -> { order(start_time: :desc) }
+
+  enum :status, {
+    not_started: 'not_started',
+    in_progress: 'in_progress',
+    success: 'success',
+    failed: 'failed',
+  }, prefix: true
+
+  def similar_persons
+    potential_duplicate_people.order(score: :desc)
```

Comment:
Again, this is a good candidate for a `default_scope` or at least for a scoped access directly in `has_many :potential_duplicate_people`.

Reply:
Okay, added scope in `has_many`.